superset/assets/utils/common.js

```diff
@@ -39,8 +39,19 @@ export function getParamFromQuery(query, param) {
   return null;
 }
 
-export function getLink(baseUrl, params) {
-  return baseUrl + '?' + params.join('&');
+export function getQueryLink(query, callback) {
```

Comment:
it will be cleaner to have interface:
getQueryLink( dbId, title, schema,sql, callback) as you do query object initialization 2ce  to pass the same params to the function.